src/box/lua/space.cc

```diff
@@ -453,7 +453,6 @@ lbox_fillspace(struct lua_State *L, struct space *space, int i)
 
 		lua_pushstring(L, "parts");
 		luaT_push_key_def(L, index_def->key_def);
-
```

Comment:
Nit: It is the only change in the file :)

Reply:
Yes, that empty line was a bit misleading.

Reply:
Hm. OK.